pykit/transport/http/HttpConnection.py

```diff
@@ -0,0 +1,63 @@
+import requests
+from ...Toolkit import Toolkit
+
+
+class HttpConnection:
+    """
+    Represents a REST HTTP connection that can be used to get the Python
+    Toolkit object.
+    """
+
+    def __init__(self, db2sock_rest_url, db2sock_auth):
+        self.db2sock_auth = db2sock_auth
+        self.db2sock_rest_url = db2sock_rest_url
+        self.payload = []
+        self.test_db2sock()
+
+    def toolkit(self):
+        """
+        Return an instance of the toolkit with a connection defined.
+
+        :return: Toolkit
+        """
+        return Toolkit(self)
+
+    def add(self, o):
+        """
+        Add an object to the payload that will be passed to DB2Sock.
+
+        :param o: Object to be added
+        :return: void
+        """
+        if isinstance(o, dict):
+            self.payload.append(o)
+        else:
+            raise TypeError('Only dictionaries are supported right now.')
+
+    def execute(self):
+        """
+        Execute the payload and then clear the payload.
+
+        :return: Response
+        """
+        response = requests.post(
+            self.db2sock_rest_url, json=self.payload, auth=self.db2sock_auth)
+        self.payload = []
+        return response
+
+    def test_db2sock(self):
```

Comment:
Do we envision this to be a public API or just an internal one? If not public, it should be prefixed with two underscores to prevent it from being called directly (but don't use trailing underscores, since those names are reserved by python itself). eg. `test_db2sock` -> `__test_connection`